cmake/ctest/drivers/atdm/tlcc2/drivers/Trilinos-atdm-tlcc2-intel-debug-openmp-panzer.sh

```diff
@@ -3,4 +3,4 @@ export SALLOC_CTEST_TIME_LIMIT_MINUTES=0:45:00
 if [ "${Trilinos_TRACK}" == "" ] ; then
   export Trilinos_TRACK=Specialized
 fi
-$WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/chama/local-driver.sh
+$WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/tlcc2/local-driver.sh
```

Comment:
@fryeguy52, I think it would be better to also change the names of these scripts from "chama" to "tlcc2".  I think it would be good to keep the convention where the ATDM system type ('tlcc2' in this case) is in the build name and the actually machine this build is run on is given in the site name on CDash.